### DISCUSSION TITLE
How to Configure Discount % or Discount Rate at Product Level

### DISCUSSION THREAD

**Discussion by ramkishore-yerubandi** [vendurehq/vendure#1380] [Q&A] [answered]
Is there any option to set Discount on Product/Variant without setting Promotions? 

**The goal is to** 
1. Offer discounts at the product level by default to all customers without applying coupon code by customers.
2. Show discounted prices to customers at the storefront and display the savings.

**Comment by michaelbromley** [comment, marked as answer]
For this I would recommend you set up a custom field on ProductVariant with "RRP" (full price) and keep you "discounted" price as the actual price.

**Reply by dachieu1996** [reply]
Hi, discounted price should insert on `ProductVarientPrice` entity, but from now I can't create custom fields for this entity. How can I add a discount price for a product variant?
